tethys_apps/base/controller.py

```diff
@@ -36,6 +37,9 @@
 from typing import Union, Any
 from collections.abc import Callable
 
+global_page_controller = optional_import(
```

Comment:
Why is this an `optional_import`?  Since it's part of the source code we should be able to import it directly right?

Reply:
I debated a bit on how I should implement this. It's optional because the tethys_apps.base.page_handler module requires that `reactpy` (the true optional import) be imported at the top level since one of the importable functions itself has a `reactpy` decorator. So importing anything from `tethys_apps.base.page_handler` would fail if `reactpy` was not installed. But I just dug into it again, and realized that I actually updated the `reactpy` import in `tethys_apps.base.page_handler` itself to have optional behavior. So I think you're right that we ultimately don't need the optional behavior here anymore. I'll get that cleaned up.